### PR TITLE
Dbsetting part3

### PR DESCRIPTION
**Pull request by Tomoki108**
***Outline***
word/word_answer/lesson_word models and migration files created. 

***Commands to run***
- php artisan migrate

***Precondition***
- PR#2 (Dbsetting part2) is already merged

***Expected out put***
- word/word_answer/lesson_word models and migration files is created. And those 3 databases also would be created properly by the command.

### CODE REVIEW

File: app/Lesson_word.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Lesson_word extends Model
```
Comment (jeremiahC) [inline app/Lesson_word.php]: Please don't use underscore For class name, we use camel case e.g LessonWord

File: app/Word_answer.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Word_answer extends Model
```
Comment (jeremiahC) [inline app/Word_answer.php]: Please follow the comments above

File: app/Lesson_word.php (hunk 1 above)
Comment (jeremiahC) [reply app/Lesson_word.php]: Please follow Coding style standards base from this article
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md

File: app/LessonWord.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Lesson_word extends Model
```
Comment (jeremiahC) [inline app/LessonWord.php]: Not only the file name should be changed also the class name, but class name and file name should also be the same.

Comment (Tomoki108) [reply app/LessonWord.php]: Sorry I was careless. It's re-fixed.

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**